sideboard/config.py

```diff
@@ -38,36 +99,64 @@ def get_dirnames(pyname, plugin):
     package under the configured "plugins_dir" directory.
 
     These two directories are also automatically inserted into plugin config
-    files as "root" and "module_root" and are available for interpolation.  For
-    example, a plugin could have a line in their config file like
+    files as "root" and "module_root" and are available for interpolation. For
+    example, a plugin could have a line in their config file like::
```

Comment:
I believe those are full colons.  A lot of programming languages use full colons when only a semicolon would do :P